sensu/transport/rabbitmq/rabbitmq.go

```diff
@@ -4,19 +4,60 @@ import (
 	"errors"
 	"fmt"
 	"math/rand"
+	"strconv"
 	"time"
 
 	"github.com/upfluence/sensu-go/Godeps/_workspace/src/github.com/streadway/amqp"
 	"github.com/upfluence/sensu-go/Godeps/_workspace/src/github.com/upfluence/goutils/log"
 )
 
+// AmqpChannel is an interface over amqp.Channel
+type AmqpChannel interface {
```

Comment:
because amqp is an acronym, the Go styleguide recommend to name it all in capital, `AMQPChannel`